src/main/kotlin/no/nav/tiltakspenger/saksbehandling/meldekort/domene/MeldekortUnderBehandling.kt

```diff
@@ -136,6 +138,22 @@ data class MeldekortUnderBehandling(
         return KunneIkkeUnderkjenneMeldekortBehandling.BehandlingenErIkkeKlarTilBeslutning.left()
     }
 
+    override fun overta(
+        saksbehandler: Saksbehandler,
+    ): Either<KunneIkkeOvertaMeldekortBehandling, MeldekortBehandling> {
+        return when (this.status) {
+            UNDER_BEHANDLING -> {
+                this.copy(
+                    saksbehandler = saksbehandler.navIdent,
+                ).right()
+            }
+            KLAR_TIL_BESLUTNING,
+            GODKJENT,
+            -> throw IllegalStateException("Meldekort under behandling kan ikke ha status ${this.status}")
+            IKKE_RETT_TIL_TILTAKSPENGER -> KunneIkkeOvertaMeldekortBehandling.BehandlingenKanIkkeVæreGodkjentEllerIkkeRett.left()
```

Comment:
Kan sikker kaste på denne også. Frontend bør sperre for å kunne komme i den situasjonen.